### DISCUSSION TITLE
How is the performance on a Raspberry Pi?

### DISCUSSION THREAD

**Discussion by TheOtherUnknown** [blakeblackshear/frigate#8412] [Ask A Question] [answered]
Hi all,

I'm considering hardware for a new Frigate setup, and I noticed that the docs briefly mention Raspberry Pi support. Given the limited specs of a Raspberry Pi, I'm curious how the app performs on the platform compared to say, an Intel NUC. 

Additionally, I know that the latest Pi, the Pi 5, drops support for H264 hardware acceleration. I think this may have an impact on performance. 

With that in mind, does anyone have any performance impressions or numbers they can share?

**Comment by NickM-27** [comment, marked as answer]
Pi4 was dropped from recommended hardware, some people use it with a couple cameras but in general I think there are better options that are the same or slightly higher price.

**Reply by twister36** [reply]
Yes - I'm running a Pi4 with 4 cameras, but I get plenty of warnings. Do we have a recommended hardware? I'm looking to replace my infrastructure.

**Reply by dz4news** [reply]
I have been using a Micro PC that I picked up on eBay for $200 USD. I wiped the OS and installed Ubuntu and run frigate in docker with a coral.
The system has been rock solid running 6 cameras. I am sure I could add a few more without any issues.
Mine is the Dell Micro Form Factor Optiplex

**Reply by twister36** [reply]
Are you recording video clips? I didn't have any issues until I started to record.

**Reply by dz4news** [reply]
Yes, on all 6. I have record set for 4 seconds before and after events within specific zones for each camera.

**Reply by twister36** [reply]
Odd... even though my recording work, the logs are full of errors reporting it can't keep up with the stream or it abandoned recordings... I suspect due to the hardware not being able to keep up with the stream. I'm migrating to a desktop for my instance.

**Comment by dataslayermedia** [comment]
Frigate is pretty savage on my Raspberry Pi 5 with Coral PCIe accelerator...
https://youtu.be/so8_h0EKPWU
